schema/record-package-schema.json

```diff
@@ -99,7 +99,11 @@
         "type": "string",
         "format": "uri"
       },
-      "uniqueItems": true
+      "uniqueItems": true,
+      "deprecated": {
+        "description": "This field hadn't clear and real use cases.",
```

Comment:
```suggestion
        "description": "This field is deprecated, because it is rarely implemented, is rarely used, and depends on a publication pattern in which release packages are published at stable URLs.",
```